CDP4CrossViewEditor/Views/UserControls/ThingUserControl.cs

```diff
@@ -54,14 +56,35 @@ public ClassKind Mode
             set => this.SetValue(ModeDependencyProperty, value);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ThingUserControl"/> class.
+        /// </summary>
+        protected ThingUserControl()
+        {
+            this.DataContextChanged += this.ThingUserControl_DataContextChanged;
```

Comment:
Isn"t behavior best for event handler?

Reply:
I don't know if this is the best but at least here it's needed for binding different data to ElementThingSelector & ParameterThingSelector user controls.